lib/tc/request.go

```diff
@@ -22,23 +22,31 @@ func (s Server) makeGetRequest(endpoint string) (int, string, error) {
 }
 
 func (s Server) makePostRequest(endpoint, body string) (int, string, error) {
```

Comment:
this should be renamed now to
```suggestion
func (s Server) makePostRequestWithXMLContentType(endpoint, body string) (int, string, error) {
```
?